templates/analytics/realm_details.html

```diff
@@ -1,11 +1,11 @@
 <span class="label">realm</span>
-<h3><img src="{{ realm.realm_icon_url }}" class="support-realm-icon"> {{ realm.name }}</h3>
+<h3><img src="{{ realm_icon_url(realm) }}" class="support-realm-icon"> {{ realm.name }}</h3>
 <b>URL</b>: <a target="_blank" href="{{ realm.uri }}">{{ realm.uri }}</a> |
 <a target="_blank" href="/stats/realm/{{ realm.string_id }}/">stats</a> |
 <a target="_blank" href="/realm_activity/{{ realm.string_id }}/">activity</a><br>
 <b>Date created</b>: {{ realm.date_created|timesince }} ago<br>
-<b>Admins</b>: {{ realm.admin_emails }}
-<a title="Copy emails" class="copy-button" data-admin-emails="{{ realm.admin_emails }}">
+<b>Admins</b>: {{ realm_admin_emails(realm) }}
```

Comment:
I think it doesn't make sense to do function computations in templates -- we should just pass in the results of realm_admin_emails(realm), etc.

Reply:
@rishig This is my reasoning behind this change. 

We currently have code to calculate the value of realm_icon_url, admin_emails and default_discount in two diffrent places. With the addition of showing confirmation links it would become three. The easiest way to deduplicate the code and make the view cleaner is by doing the calculations in template. Alternatively one can write a function that takes users, realms and confirmations as arguments and sets the value of realm_icon_url, admin_emails and default_discount appropriately in realm object according to the type of the confirmation. But that seems more messy than passing the functions directly to template approach.